src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,67 @@
+package mate.academy.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import java.time.LocalDateTime;
+
+@Entity
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false)
+    private Long id;
+    @ManyToOne
+    @JoinColumn(name = "movie_id")
+    private Movie movie;
+    @ManyToOne

```

Comment:
I`m not sure, but not one cinema hall can have one session?

Reply:
The field cinemaHall in the MovieSession entity is annotated with @ManyToOne, indicating a many-to-one relationship between the MovieSession and CinemaHall entities. This means that one cinema hall can have many movie sessions (MovieSession), but each movie session belongs to only one cinema hall (CinemaHall).